core-bundle/src/Resources/contao/library/Contao/Environment.php

```diff
@@ -190,7 +190,7 @@ protected static function queryString()
 	}
 
 	/**
-	 * Return the request URI [path]?[query] (e.g. /contao/index.php?id=2)
+	 * Return the request URI [path]?[query] (e.g. /index.php?id=2)
```

Comment:
This is wrong. The method does return the path.

Reply:
no idea what it returns, but it certainly does not return `/contao/index.php?id=2` 😂

Reply:
Are you sure? It would only do that, if you access Contao via `localhost` - or may be when aliasing the `public` directory.

Reply:
you cannot/should not run Contao in a subfolder. This return example refers to our old entry point, certainly not a setup in subfolder `contao`. Also, this is just an example what the method returns, it really does not matter 😂

Reply:
I don't get this discussion. The changed line is clearly a **comment example** of an entry point script that no longer exists. 🤷 Yes you can technically create a local folder `contao` and run the `index.php` from there, but this is **obviously** not what the comment was about.